### DISCUSSION TITLE
怎么添加非沙拉查词扩展自带的新词典 How to add a new dictionary that saladict doesn't possess

### DISCUSSION THREAD

**Discussion by playermiller109** [crimx/ext-saladict#2196] [Q&A]
I hope to add a new dictionary by myself: https://www.thesaurus.com

The composition of the dictionary is very simple. Using {query} to represent the site address search term, the word search URL is https://www.thesaurus.com/browse/{query}

How can I finish it? Any help would be appreciated.

Chinese version of this comment:
本评论中文版：

我希望自行添加一个新词典：https://www.thesaurus.com

这个词典的构成非常简单，以 {query} 表示站点地址搜索词，查词网址为 https://www.thesaurus.com/browse/{query}

我该如何操作？谢谢。

**Comment by crimx** [comment]
目前不支持用户加入词典
